cip-0079/cip-0079.md

```diff
@@ -0,0 +1,118 @@
+* **Phase 3: Official CC-USD BRR, underlying ISIN and regulatory supervision**
+    * Oversight committee will have to vote the transition to phase 3
+    * Kaiko will be required to have an ISIN issued for the CC-USD BRR
+    * The CC-USD BRR will have to fall under regulatory supervision of Kaiko's regulated indices 
+
+## **Appendix IV - Upgrading the Onchain Price Technical Dissemination**
+
+Super Validators will no longer have to hardcode the CC/USD conversion rate but will instead have the ability to access it on-chain using Kaiko's on-chain feeds within the Canton Network. Super Validator can access across all phases to the CC-USD BRR through Kaiko's On-ramp facility and on-chain price feeds, ensuring that there will be no technical implication of the approach. 
+
+The implementation of the underlying infrastructure does not imply dependencies with the above approach, except for phase 1 price input. During the initial phase, Kaiko will have to expose a price collection facility allowing SVs to submit publicly their prices in order to replicate the current price formation mechanism at the time of writing of this CIP.
+
+**The revised process will be as follows:**
+
+_**Specific to phase 1: Replicating current process**_
+
+* CC/USD Price contributions from SVs
+     * Kaiko will collect the CC/USD prices from Canton through an interface accessible to all Super Validators. SVs will have the ability to change their prices for the next publication. If no changes are submitted by an SV, their previously submitted value will be used.
+
+* CC/USD Price calculation from SVs
+    * Once Super Validators have uploaded their prices to Kaiko, the median price will be calculated off-chain by Kaiko, replicating SPLICE methodology, and Kaiko will publish the median price back on the Network.
+    * In subsequent phases, the price calculation will no longer rely on SV price input but the CC-USD rate will be a Mark-to-Market rate. As such, SVs will only be required to choose the source, and the downstream process and integration with SPLICE and AMULET will be fully automated.
+
+* Generic: publishing the CC-USD price 
+    * The price feed will be published on-chain via Kaiko's Canton Price feeds on the network, initially every 1 min, then eventually in the future up to second/real-time level updates if required or beneficial. The technical price format, Data contract standard shall match the standard of the overall facility, and be maintained by Kaiko with that regard. The technical interfaces must not be price discovery phases dependent. SVs can then use the Kaiko CC-USD price to programmatically (vs manually at the time of writing) submit their price to SPLICE, or choose another on-chain source. The DSO will publish the consensus price from SPLICE to AMULET, regardless of their sources. SVs will thus be able to choose to rely on the Kaiko CC-USD BRR directly into SPLICE (vs manual update). They could also choose to use another source.
```

Comment:
The Daml model sketch from https://github.com/hyperledger-labs/splice/pull/1980 only supports updating the price every 5min. Is the plan to publish it indepentently of that at a higher frequency?